activiti-cloud-notifications-graphql-service/services/events/src/main/java/org/activiti/cloud/services/notifications/graphql/events/consumer/EngineEventsConsumerMessageHandler.java

```diff
@@ -44,25 +44,25 @@ public EngineEventsConsumerMessageHandler(Transformer transformer,
         this.transformer = transformer;
     }
 
-    @StreamListener
-    public void receive(@Input(EngineEventsConsumerChannels.SOURCE)
-                            Flux<Message<List<Map<String,Object>>>> input) {
+    @StreamListener(EngineEventsConsumerChannels.SOURCE)
+    public void receive(Message<List<Map<String, Object>>> input) {
```

Comment:
Unfortunately, EnableBinding and functional are mutually exclusive. In the future, we should migrate to the functional approach, even because annotation-driven is all but deprecated.